utils/supported-browsers-from-mdn.mjs

```diff
@@ -12,7 +13,10 @@ const MDNToBrowserlist = {
 
 function getBrowsersFromFeature(mdnConfigPath, feature) {
 	const mdnFeature = _get(mdn, mdnConfigPath);
-	const supportsPrefixes = feature.allow_partial_implementation;
+
+	// We assume users also have autoprefixer.
+	// If autoprefixer adds prefixes for this feature we count it as supported.
+	const supportsPrefixes = Object(feature.mdn_count_prefixed_as_supported)[mdnConfigPath];
```

Comment:
Per sub feature escape hatch to allow prefixes.
This is granular because we need to align exactly with what exists in autoprefixer